changelogs/unreleased/5129-pnbrown-docs

```diff
@@ -0,0 +1 @@
+Upgrade algolia docsearch to v3 on the docs website
```

Comment:
Looks like you need to add a `.md` extension to the filename

Reply:
Yup. Just caught that. My bad.